src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -590,6 +611,11 @@ public void Dispose()
 
         internal static CommonSecurityDescriptor GetServerPipeSecurity()
         {
+            if (!Platform.IsWindows)
```

Comment:
Why runtime check? We could use `#if UNIX`.

Reply:
I think originally I left this to see if this method made sense on UNIX, but it's for allowing admins to enter a user's runspace.  For UNIX, I think we should just only allow same user.  I'll update the code to `#if`.